src/common/settings.h

```diff
@@ -505,6 +505,7 @@ struct Values {
     SwitchableSetting<bool> upright_screen{false, "upright_screen"};
     SwitchableSetting<float, true> large_screen_proportion{4.f, 1.f, 16.f,
                                                            "large_screen_proportion"};
+    SwitchableSetting<float> screen_gap{0.f,"screen_gap"};
```

Comment:
If `screen_gap` is supposed to be an int, then this should be an int rather than a float.

Reply:
ah fair, i'll fix that soonish